package.json

```diff
@@ -38,17 +38,22 @@
       "js"
     ],
     "moduleNameMapper": {
-      "^@taquito/rpc$": "<rootDir>/packages/taquito-rpc/src/taquito-rpc.ts",
-      "^@taquito/taquito$": "<rootDir>/packages/taquito/src/taquito.ts",
+      "^pack-test-tool$": "<rootDir>/packages/taquito-michel-codec/pack-test-tool/src/pack-test-tool.js",
```

Comment:
curious what's pack-test-tool and why added it in moduleNameMapper

Reply:
I just made sure that all packages that are part of the npm workspace are added here.